.github/workflows/frontend-ci.yml

```
@@ -0,0 +1,130 @@
+    - name: Run type check
+      working-directory: ./frontend
+      run: npx tsc --noEmit
+    
+    - name: Run tests
+      working-directory: ./frontend
+      run: npm test -- --coverage --watchAll=false
+      env:
+        CI: true
+    
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)